### ISSUE TITLE
May 2024

### ISSUE THREAD

**Issue by kawogi** [Rustaceans/rust-cologne#117]
# ToDo

- [x] Settle for a date and location: **Wednesday 2024-05-01, 19:15 CEST** at C4 (default: first Wednesday each month)
- [ ] Send "Save the Date" info (with link to this ticket)
- [x] Basic Agenda (default: Open Space)
- [x] Final invitation (deadline **2023-04-24 T - 1 week**)
  - [x] [meetup](https://www.meetup.com/rustcologne/events/300610856/)
  - [x] [website](https://rust.cologne/2024/05/01/this-month-in-rust.html)
  - [ ] [twitter]
  - [x] [community calendar]

# Invitation draft

## This Month In Rust, May

_(The meetup will likely be held in German but we'll switch to English if needed.)_

Dear Rustacean,

our next Meetup will be on **Wednesday 2024-05-01, 19:15 CEST**.

You're new to Rust, tried it out with some private projects or are already working professionally with it for years? You're in good company! Whether you want to speed up your python code, write a web service, need interop with your C++-codebase, cross-compile for a microcontroller, write an emulator, … we've got you covered!

We commonly open up with a brief summary of noteworthy changes in the Rust ecosystem. Be it updates to the language itself, public events, its impact on other projects and languages, …

After that we often have volunteers talking about specific topics or personal projects, but in general the remaining time is about whatever _you_ want to talk about!

Always wanted to know why Rust is harder to learn than other popular languages or why it lacks a certain feature? When is it appropriate to rewrite a project in Rust? Is the compiler really that slow and are the binaries really that big? Do all those safety guarantees have an impact on the performance? Fetch a drink and let's find it out.

**[You can register here](...)**.

See you soon!

Yours,
Florian and Kai
- - -
If you have a topic you'd like to talk about, please [let us know in advance](…). This way we can make sure there's a time slot for you and maybe announce it officially. Thank you.

# Rust News
- _TBD_

**Comment by kawogi** [comment]
@Florob The 1st of May is a public holiday. Shall we move to the 8th?

Any ideas for an agenda?

**Comment by nwalfield** [comment]
FYI, [rustnl](https://2024.rustnl.org/) is May 8th, which might impact the number of participants.

**Comment by Florob** [comment]
@kawogi I thought we had agreed on doing another open discussion one.
I personally don't mind doing it on a public holiday. Makes it easier to drop by for some, harder for others.
I'm not fundamentally opposed to the 8th either though. However, nwalfield is probably right about rustnl (thanks for letting us know) and I would ideally have had to confirm that date at the plenary yesterday…
TL;DR: I have a slight preference for the 1st, but feel free to make the call.

**Comment by kawogi** [comment]
Ok, let's keep the 1st and make it an open agenda :+1:
